App/GuiConfig.cpp

```diff
@@ -0,0 +1,57 @@
+//
+//    GuiConfig.cpp: Gui configuration object
+//    Copyright (C) 2019 Gonzalo José Carracedo Carballal
+//
+//    This program is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as
+//    published by the Free Software Foundation, either version 3 of the
+//    License, or (at your option) any later version.
+//
+//    This program is distributed in the hope that it will be useful, but
+//    WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public
+//    License along with this program.  If not, see
+//    <http://www.gnu.org/licenses/>
+//
+
+#include "GuiConfig.h"
+
+using namespace SigDigger;
+
+GuiConfig::GuiConfig()
+{
+  this->loadDefaults();
+}
+
+GuiConfig::GuiConfig(Suscan::Object const &conf) : GuiConfig()
+{
+  this->deserialize(conf);
+}
+
+void
+GuiConfig::loadDefaults(void)
+{
+  this->useLMBdrag = false;
+}
+
+#define STRINGFY(x) #x
```

Comment:
There is already a `STRINGIFY` (with an I between STRING and FY, I know, it's odd) with identical behavior you can use by including `<Suscan/Compat.h>`

Reply:
actually it was forgotten copy/paste from the ColorConfig.cpp ;-)
I was thinking if a template will be helpful here, but unfortunately the stringify must be still done by macro.